lib/vrl/parser/src/parser.lalrpop

```diff
@@ -255,13 +255,15 @@ AssignmentOp: AssignmentOp = {
 AssignmentSingle: Assignment = {
     <target: Sp<AssignmentTarget>>
         <op: AssignmentOp>
+        NonterminalNewline*
```

Comment:
One of the Cue doc examples used:

```rust
foo =
	a ??
	b ??
	c
```

This is now supported.

There are probably more places we want to allow non-terminal newlines, but we can iterate on that going forward.